_sass/_header.scss

```diff
@@ -131,3 +131,45 @@ header {
     }
   }
 }
+
+.dropdown__header strong {
+  margin-left: 5px;
+  margin-right: auto;
+  font-size: 1.6rem;
+}
+
+.dropdown__header .dropdown__categories,
+.dropdown__header .dropdown__menu {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.dropdown__header li {
+  padding: 10px;
+  position: relative;
+}
+
+.dropdown__menu li {
+  background: $black;
+  color: $white;
+  border: 1px solid $white;
+
+  &:hover {
+    background: $red;
+  }
+}
+
+.dropdown__header .dropdown__category .dropdown__menu {
+  display: none;
+  position: absolute;
+  background: $white;
+  width: 200px;
+  top: 50px;
+  right: 0;
+}
+
+.dropdown__header .dropdown__category:hover .dropdown__menu {
```

Comment:
the hover category here is likely too small, should more padding be added to `.dropdown__category` or should the hover pseudo selector be added elsewhere? @ovlb any ideas?